cluster/src/test/java/com/hartwig/pipeline/metadata/ResearchMetadataApiTest.java

```diff
@@ -208,6 +220,32 @@ public void doesNothingOnFailedPipeline() {
         verify(publisher, never()).publish(any());
     }
 
+    @Test
+    public void setsStatusAndStartTimeOnStart() {
+        ArgumentCaptor<Long> runIdArgumentCaptor = ArgumentCaptor.forClass(Long.class);
+        ArgumentCaptor<UpdateRun> updateRunArgumentCaptor = ArgumentCaptor.forClass(UpdateRun.class);
+        when(runApi.update(runIdArgumentCaptor.capture(), updateRunArgumentCaptor.capture())).thenReturn(run);
+        victim.start();
+        assertThat(runIdArgumentCaptor.getValue()).isEqualTo(RUN_ID);
+        UpdateRun updateRun = updateRunArgumentCaptor.getValue();
+        assertThat(updateRun.getStartTime()).isNotNull();
```

Comment:
Would the AssertJ's `isCloseTo` be appropriate here? Would definitely be more rigorous than just making sure it's non-null.